### PR TITLE
Fire add to webhook

### PR DESCRIPTION
**Pull request by rtewalt**
| Status  | Type  | Migrations
| :---: | :---: | :---: |
| Ready | Feature | No |

## Description

Will call the AddAssociation webhook when it gets an UpdateProduct message for the right store and has all the necessary retailer information in the message.

## Other changes

Adds linting to the project.

## Related Tickets

List related Clubhouse tickets:

* [Sync product association information to partners](https://app.clubhouse.io/shipt/story/48704/sync-product-association-information-to-partners)

## Todos

_Fill out or remove list items as needed_

- [x] Tests
- [x] Documentation

## Deploy Notes

N/A

### CODE REVIEW

**Comment by rtewalt** [comment]
retest this please

**Comment by rtewalt** [comment]
retest this please

File: internal/app/syncproduct/config/config.go [hunk 1]
```diff
@@ -1,44 +1,88 @@
...
+var Current *Config
+
 func ReloadConfig() {
+
 	newConfig, err := loadContextConfig()
 
 	if err != nil {
-		log.Printf("Could not load config, using existing %v", Current)
-		return
+		log.Printf("Could not load config, %v", err)
```
Comment (vadimuchitel) [inline internal/app/syncproduct/config/config.go]: @rtewalt That needs to go to Rollbar / Slack. We've had this issue multiple times before added Slack integration into go-core where Service would not start and nobody would know about it. We don't have cool Netflix tools like you guys have :-)

Comment (rtewalt) [reply internal/app/syncproduct/config/config.go]: thanks for the catch, this just a miss

**Pull request merged**